templates/_search-form.html

```diff
@@ -0,0 +1,131 @@
+<form method="GET" class="p-form js-asset-search">
+  {% set params = request.values.to_dict() %}
+  <div class="row">
+
+    <div class="col-4">
+      <label for="tag">Tag</label>
+      <input class="js-tags" type="text" name="tag" placeholder="Tag" id="tag" value="{% if params.tag %}{{ params.tag }}{% endif %}">
+      <button class="p-button--positive" style="margin-top:45px;">Search</button>
+    </div>
+
+    <div class="col-8">
+      <div class="row">
+        {% set asset_types = [
+          "Datasheets", "Guides", "Ebook", "Whitepaper", "Case Study", 
```

Comment:
```suggestion
          "Datasheet", "Guide", "Ebook", "Whitepaper", "Case Study", 
```
Maybe they should all be singular

Reply:
This comes from the spec so I will it as is for now and add comment for Sohpie